src/main.zig

```diff
@@ -3092,7 +3174,14 @@ pub fn cmdFmt(gpa: *Allocator, args: []const []const u8) !void {
     defer fmt.out_buffer.deinit();
 
     for (input_files.items) |file_path| {
-        try fmtPath(&fmt, file_path, check_flag, fs.cwd(), file_path);
+        fmtPath(&fmt, file_path, check_flag, fs.cwd(), file_path) catch |err| {
```

Comment:
This doesn't seem correct, `fmtPath` already handles everything except `AccessDenied` which should be handled in `fmtPathDir` and not be fatal.

Reply:
Good point, I missed that in my first read.